PrestaShop/PrestaShop
# Initialize full Symfony layout in Back Office

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new Symfony layout
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI should be green and so should:<br>Regular UI tests (no symfony layout option available) https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/5327467076<br>UI tests with symfony layout https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/5332273359 (still 10 campaigns failing, to be fixed later)<br>UI tests with explicit legacy layout (no regression) https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/5328644737
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### How to test
To test the full UI tests campaign a new option was added to our tool in this PR https://github.com/PrestaShop/ga.tests.ui.pr/pull/30 this adds a new input in the launch modal that allows selecting which layout the back office will run on by switching the related feature flag via a CLI command
This should be used in all future PRs related to Symfony layout

<img width="348" alt="Capture d’écran 2023-05-02 à 11 15 35" src="https://user-images.githubusercontent.com/13801017/235627904-8bcb7059-eb30-485f-a14b-352e4d3d23e1.png">

See example: https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4859492012/jobs/8662193087

### POC Result

This PR is only a POC a lot of things still need to be improved/done an Epic has been created https://github.com/PrestaShop/PrestaShop/issues/32875 to summarize all the remaining work to do.
